runtime/neumann/src/lib.rs

```diff
@@ -971,12 +990,14 @@ impl_runtime_apis! {
 			use pallet_automation_time::Pallet as AutomationTime;
 			use pallet_valve::Pallet as Valve;
 			use pallet_vesting::Pallet as Vesting;
+			use parachain_staking::Pallet as ParachainStaking;
```

Comment:
Very unclear on how to use the runtime benchmarks. Only included this here because I saw other projects doing the same.